rosdep/python.yaml

```diff
@@ -204,6 +204,7 @@ ipython:
   gentoo: [dev-python/ipython]
   macports: [py27-ipython]
   nixos: [pythonPackages.ipython]
+  opensuse: [python2-ipython]
```

Comment:
I think the links provided are for `python-ipython` (and `python3-ipython`), as opposed to `python2-ipython`. Adding links for the latter for future reference
https://software.opensuse.org/package/python2-ipython
https://build.opensuse.org/package/binaries/openSUSE:Leap:15.2/python2-ipython/standard

Could you confirm the key currently in the file is the one you want?

Reply:
Yes, that was my mistake. The key is correct. I provided the wrong links.

Reply:
Thanks!